test/e2e/apps/daemon_set.go

```diff
@@ -842,7 +842,14 @@ var _ = SIGDescribe("Daemon set [Serial]", func() {
 		framework.ExpectEqual(len(dsList.Items), 0, "filtered list should have no daemonset")
 	})
 
-	ginkgo.It("should verify changes to a daemon set status", func() {
+	/*	Release: v1.22
+		Testname: DaemonSet, validate status endpoints
+		Description: When a DaemonSet is created it MUST succeed.
+		When retrieving /status the action MUST be validated.
```

Comment:
Here too, it's not clear to me what this description means.

Reply:
Fixed